### ISSUE TITLE
layout D (previously called layout C)

### ISSUE THREAD

**Issue by ozanj** [ozanj/heoc20grads#6]
specifications
- large landscape image or text block
- row of text at top or bottom [could be one text block spanning two columns; or two separate text blocks]

CREATING THIS TEMPLATE IS LOWER PRIORITY. DO ONLY AFTER YOU HAVE DONE ALL OTHER TASKS.
